rosidl_generator_py/msg/Strings.msg

```diff
@@ -1,5 +1,9 @@
 string empty_string
 string def_string "Hello world!"
+string def_string_delimiter 'Hello"world!'
+string def_string_delimiter2 "Hello'world!"
+string def_string_delimiter3 'Hello'world!'
+string def_string_delimiter4 "Hello"world!"
```

Comment:
because `rosidl_parser` already escapes the inner quotes before feeding the strings to the generators. I thought it was the expected behavior (based on your comment [here](https://github.com/ros2/rosidl/pull/197#discussion_r94444096)).
To clarify we're talking strings and not string arrays here, the question of requiring users to escape inner quotes came up when I started to look into parsing string arrays default values